app/controllers/people_controller.rb

```diff
@@ -0,0 +1,27 @@
+class PeopleController < ApplicationController
+
+  def index
+    @all_people = Person.all
+
+    render :index
+  end
+
+  def show
+    @person = Person.find(params[:id])
+
+    @person_id =        @person[:id]
+    @person_name =      @person[:name]
+    @person_fav_genre = @person[:fav_genre]
+
+    render :person
```

Comment:
Same comment here about the `@person`  variable having all of the data you need. I'd also recommend matching your view name to the action name for clarity, i.e. renaming the person view to show. Since the views are in the related folder, `/views/people/show.html.erb` this helps us understand already that this is related to people.